tests/test_reports/test_trajectories_report.py

```
@@ -0,0 +1,517 @@
+def test_counts_and_score(bundle: Path) -> None:
+    report = json.loads(generate_trajectories_report(bundle).read_text())["benchmarks"][0]
+    traj = report["trajectories"]
+    assert traj["problems"] == 2
+    assert traj["repeats"] == 1
+    assert traj["mean_reward"] == 0.75
+
+
+def test_tokens_section(bundle: Path) -> None:
+    report = json.loads(generate_trajectories_report(bundle).read_text())["benchmarks"][0]
+    t = report["tokens_stats"]
+    # 2 trials × (pt+ct): trial0=15, trial1=20+10+30+8=68 → 83 across all
```

Ruff (0.15.17)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)